x/staking/types/params.go

```diff
@@ -208,3 +225,19 @@ func ValidatePowerReduction(i interface{}) error {
 
 	return nil
 }
+
+func validateMinCommissionRate(i interface{}) error {
+	v, ok := i.(sdk.Dec)
+	if !ok {
+		return fmt.Errorf("invalid parameter type: %T", i)
+	}
+
+	if v.IsNegative() {
+		return fmt.Errorf("minimum commission rate cannot be negative: %s", v)
+	}
+	if v.GT(sdk.OneDec()) {
+		return fmt.Errorf("minimum commission rate cannot be greater than 100%%")
```

Comment:
```suggestion
		return fmt.Errorf("minimum commission rate cannot be greater than 100%: %s", v)
```

Reply:
I believe we need the double percent sign to print a percent sign.

https://programming.guide/go/fmt-printf-percent-sign.html

Reply:
But I'll add the printed value

Reply:
But I'll add the printed value

Reply:
```suggestion
		return fmt.Errorf("minimum commission rate cannot be greater than 100%%: %s", v)
```